src/classes/BDI_DataImportService.cls

```diff
@@ -962,7 +962,7 @@ global with sharing class BDI_DataImportService {
 
         // now update the Contacts to create their Affiliations
         if (mapConIdToConUpdate.size() > 0) {
-            update mapConIdToConUpdate.values();
+            UTIL_DMLService.updateRecords(mapConIdToConUpdate.values());
```

Comment:
Also the class BDI_ContactService does all of BDI's inserting and updating of Contacts.  I assume we would want all of its saving methods to go thru UTIL_DMLService.

Reply:
Thanks for pointing out this class.